include/deal.II/grid/reference_cell.h

```diff
@@ -1794,33 +1789,31 @@ ReferenceCell::face_to_cell_lines(const unsigned int  face,
         }
       case ReferenceCells::Line:
         {
+          const auto [face_orientation, face_rotation, face_flip] =
+            internal::split_face_orientation(combined_face_orientation);
+
           return GeometryInfo<1>::face_to_cell_lines(
-            face,
-            line,
-            Utilities::get_bit(face_orientation, 0),
-            Utilities::get_bit(face_orientation, 2),
-            Utilities::get_bit(face_orientation, 1));
+            face, line, face_orientation, face_flip, face_rotation);
         }
       case ReferenceCells::Triangle:
         {
           return face;
         }
       case ReferenceCells::Quadrilateral:
         {
+          const auto [face_orientation, face_rotation, face_flip] =
+            internal::split_face_orientation(combined_face_orientation);
+
           return GeometryInfo<2>::face_to_cell_lines(
-            face,
-            line,
-            Utilities::get_bit(face_orientation, 0),
-            Utilities::get_bit(face_orientation, 2),
-            Utilities::get_bit(face_orientation, 1));
+            face, line, face_orientation, face_flip, face_rotation);
```

Comment:
We should probably replace these uses of `GeometryInfo`, but that can wait for a later time.